src/lxc/conf.c

```diff
@@ -1625,8 +1625,15 @@ static const struct id_map *find_mapped_nsid_entry(struct lxc_conf *conf,
 static int lxc_setup_devpts(struct lxc_conf *conf)
 {
 	int ret;
-	char default_devpts_mntopts[] = "gid=5,newinstance,ptmxmode=0666,mode=0620";
+	char **it;
 	char devpts_mntopts[256];
+	char default_devpts_mntopts[256] = "gid=5,newinstance,ptmxmode=0666,mode=0620";
+	char *mount_options[4] = {
```

Comment:
I think we can skip the initialization here since it's filled in single block anyway, the last `NULL` can also be added to the assignments below.

Reply:
Yup, done.